lib/localeapp/cli/install.rb

```diff
@@ -79,13 +81,25 @@ def write_config_file
           write_rails_config
         end
 
+        def write_apikey_to_dotenv
+          @output.puts "NOTICE: API key saved to .env"
+          File.open(".env", "a") do |file|
+            file.puts
+            file.puts "LOCALEAPP_API_KEY=#{self.key}"
+          end
+        end
+
+        def config_api_key_str
+          apikey_to_dotenv ? "ENV['LOCALEAPP_API_KEY']" : "'#{key}'"
```

Comment:
I think `apikey_to_dotenv` will always return `nil` here.
So we could use `#key` everywhere you changed for `#config_api_key_str` and remove this method.
Or did I miss something?